pkg/client/request.go

```diff
@@ -443,6 +449,252 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
+func (r *Request) ExecuteRemoteCommand(stdin io.ReadCloser, stdout, stderr io.Writer, tty bool) error {
+	var (
```

Comment:
Rather than pre-declaring at the top, can you move these down closer to their first usage?  (and also perhaps not declare them as a group?)